tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh

```diff
@@ -138,9 +136,7 @@ done
 
 for vol in "$POOL/vol" "$POOL/$FS/vol" ; do
 	rand_set_prop $vol checksum "on" "off" "fletcher2" "fletcher4" "sha256"
-	rand_set_prop $vol compression "on" "off" "lzjb" "gzip" \
-		"gzip-1" "gzip-2" "gzip-3" "gzip-4" "gzip-5" "gzip-6"   \
-		"gzip-7" "gzip-8" "gzip-9" "zstd"
+	rand_set_prop $vol compression "on" "off" "lzjb" "gzip" "lz4" "zle" "zstd"
```

Comment:
Maybe i'm going to sound like an extreme nitty douche now, but "on" is (always) supposed to be linking to the default level of another algorithm... As all default levels are listed, "on" doesn't have to right?

And is "zle" required in this test? I know it was missing, but was that ommision on purpose or accidental?